src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,29 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding-bottom: 24px;
+
+  &__block {
+    display: flex;
+  }
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+    background: url(../images/star.svg) no-repeat center;
+  }
+
+  &__block > .stars__star:nth-child(-n + 4) {
```

Comment:
4 [CHECKLIST] - Make sure to reuse part of [Stars task](https://github.com/mate-academy/layout_stars) here, and keep it operational in case you change block modifier.
[TASK] - Reuse stars block from [Stars task](https://github.com/mate-academy/layout_stars) and keep it operational. If you change stars--4 modifier to stars--2 or other - number of selected stars should change correctly.
Learn more [here](https://mate.academy/learn/css-advanced/sass?section=video&videoId=4944)